src/Symfony/Component/Intl/Data/Generator/LanguageDataGenerator.php

```diff
@@ -209,7 +209,7 @@ private function generateAlpha2ToAlpha3Mapping(ArrayAccessibleResourceBundle $me
 
         foreach ($aliases as $alias => $data) {
             $language = $data['replacement'];
-            if (2 === \strlen($language) && 3 === \strlen($alias) && 'overlong' === $data['reason']) {
+            if (2 === \strlen($language) && 3 === \strlen($alias) && \in_array($data['reason'], ['overlong', 'bibliographic'], true)) {
```

Comment:
seems irrelevant

we should always point alpha2 to the T code varaint

Reply:
reverted